Software-Engineering.md

```diff
@@ -133,6 +133,29 @@ _We avoid distinguishing too much between the Junior Software Engineer and Softw
 - I can confidently and charismatically pitch ideas, positively influencing my team to take decisive action based on reaching consensus.
 - I am an adept communicator, and can effectively steer technical and non-technical conversations to positive outcomes over any medium.
 
+## Principal Software Engineer
+
+> I am an influential Engineering [leader](Leadership.md) who identifies problems and opportunities relevant to Octopus' strategy, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus.
+
+### I uncover important problems and opportunities that are critical to the long-term success of Octopus
+
+- I have a deep understanding of Octopus products and strategy
+- I collaborate with other Engineering leaders to ensure I am aware of all major initiatives at Octopus and can account for them in my own initiatives
+- I have a track record of identifying significant problems, inhibitors and opportunities, and for creating lasting, best-fit solutions that align our strategy and Engineering efforts
+- I work closely with product, engineering, and executive stakeholders to articulate and refine solutions to problems that are timely, important, and impactful. 
+- My contributions go beyond patching or extending existing solutions, which may not be fit for purpose anymore. They open up whole new areas of strategic possibility that progress our journey of making Octopus a world-class Engineering organization.
+
+### I am the custodian of an area of strategic importance to Octopus
+
+- I am comfortable taking on large-scale greenfields and brownfields initiatives (e.g. re-architecting Octopus Server to allow for adding new step types as modular extensions, or creating a completely new Octopus Cloud platform)
+- I identify people and teams with the necessary skills, work out how to fill gaps, and guide and influence these people and teams collaboratively
+- I use a variety of tools and techniques to effectively communicate short and long-term plans and goals to my teams, stakeholders, Octopus senior leadership, and the broader organization
+- I use my experience to collaboratively steer teams toward good long-term technical decisions
+- The results of the initiatives I champion are measurable
+
+### I share my knowledge both within Octopus, and with the Octopus and wider developer community
+
+- I actively blog about the lessons we are learning as we improve engineering excellence to clarify our thinking, reinforce the learning, attract customers and potential future employees
```

Comment:
I want to add two new examples/points, one about an internally-focused impact as a partner to external (blogging), and one about expertly tailoring communication to the target audience. To me, these seem like key differentiators of PSE from LSE.

```suggestion
- I actively influence engineering teams across Octopus resulting in improved engineering excellence and overall productivity
- I actively blog about the lessons we are learning as we improve engineering excellence to clarify our thinking, reinforce the learning, attract customers and potential future employees
- I have a proven track record of expertly tailoring my communication to the target audience with a result of increased clarity and impact

```